wiki/Tournaments/Official_support/en.md

```diff
@@ -0,0 +1,166 @@
+
+You may be required to delay the start of your tournament during periods of high delay, but the osu!team will make efforts to ensure this does not happen.
+
+Please read the criteria laid out below, then send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh) with the following information:
+
+- The name of your tournament
+- A brief description of your tournament, including its scoring type and any other appropriate features
+- Any applicable dates for your tournament
+- Any links to public off-site chatrooms or servers used for the event (Discord, etc)
+- A link to the forum thread on the osu!forum that announces/details your tournament
+- Any links to previous iterations of the same tournament (only where applicable)
+- A list of the players involved in running the tournament
+- A list of players who have registered for the tournament
+
+Please allow at **least** a week for a response. We will try our best to handle all requests within a week, but we occasionally have large spikes in workload that can make this impossible.
+
+### Criteria
+
+Community-run tournaments which abide by the following hard criteria are eligible for support:
+
+- The tournament must have at least 64 teams/players minimum into the qualifier rounds, **OR** operate on a Round of 16 double elimination at a minimum. Larger scales and formats are permitted.
+- The tournament is organized and run by an *experienced team of accomplished and reputable volunteers***¹** , **OR**, the tournament *has been run at least once before without receiving rewards*.
+- Any of the staff members involved in the managing or organization of the tournament must not be participating in the tournament themselves at any point.
+    - To clarify further, if you play in a given tournament at all, you are unable to later assist as a referee, mappooler or anything similar until that instance of the tournament concludes. You either play or help run, not both.
+- The tournament must be run only four or fewer times per year.
+- The tournament must have all players screened by the osu! Support Team **before play commences**.
+- The tournament must not be "self-reffed".
+    - A dedicated referee must be present during every match.
+- The forum thread for the tournament must contain clearly visible link in normal-sized font to [this reporting form](LINK GOES HERE) as the very last content of the original post.
+- The tournament must not prevent players who pass screening from participation without both ample evidence presented publicly against them, and the approval of the osu! Support Team. This only applies in circumstances where they would otherwise meet the sign-up criteria for the tournament.
+    - This includes preventing users who are perceived to be 'sandbagging' from play. If this is an issue which you believe affects your tournament, consider altering your sign-up criteria or rank range.
```

Comment:
I don't quite understand what this section is aiming for. If a tournament listed "not a sandbagger" (whatever that means) in their player sign-up criteria, would they be then abiding by this section of the official support criteria?

And what about invitational tournaments? Are those still eligible for official support like they have been so far, despite seeminlgy being in direct violation of this new section? Or is the invitational fromat considered just another, very strict form of sing-up criteria?

Reply:
Apologies for not answering to your question directly, but I wanted to ask, and since it's about the same line.
In regards to sandbaggers as a whole. Does this mean hosts, or a team of independent players, are entirely denied the possibility to deny sandbaggers from their tournament, even if it's before the registration phase ends, or do we have to send a list of players with proper proof they are sandbagging in the mode they're playing for to osu!support?
This part feels really unclear to me.

Reply:
> If a tournament listed "not a sandbagger" (whatever that means) in their player sign-up criteria, would they be then abiding by this section of the official support criteria?

unless you somehow get a committee that can come to a concrete conclusion of what makes someone a sandbagger/deranker, these tournaments are way too host biased to be allowed a badge imo


> And what about invitational tournaments? Are those still eligible for official support like they have been so far, despite seeminlgy being in direct violation of this new section? Or is the invitational fromat considered just another, very strict form of sing-up criteria?

i can see the latter being an issue but assuming you're referring to RMOT, an invitational like that should be completely supported like normal